prime-router/metadata/fhir_mapping/hl7/extension/ExtensionMeta.yml

```diff
@@ -1,42 +1,42 @@
-extension_1:
+security:
```

Comment:
The Diff on this file ia a _mess_! My understanding is that it's removing some overloads that the library now handles. Can you confirm?

Reply:
It is no longer needed. That data is now in some resource property.